addons/website/views/snippets/s_references_social.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_references_social" name="References Social">
+    <section class="s_references_social o_cc o_cc1 pt64 pb64">
+        <div class="container">
+            <div class="row">
+                <div>
+                    <h2 style="text-align: center;">Our valued partners</h2>
+                    <p class="lead" style="text-align: center;">We are in good company.</p>
+                </div>
+                <div class="col-lg-3 pt16 pb16">
+                    <p style="text-align: center;">
+                        <img class="img img-fluid" src="/web/image/website.s_reference_demo_image_1" alt=""/>
+                    </p>
+                    <h3 class="h5-fs" style="text-align: center;">Amsterdam</h3>
+                    <p style="text-align: center;">Fruitful collaboration since 2014</p>
+                    <div class="s_social_media o_not_editable text-center no_icon_color" data-snippet="s_social_media" data-name="Social Media">
+                        <h4 class="s_social_media_title d-none o_default_snippet_text">Social Media</h4>
```

Comment:
Not sure we need this title, see: https://github.com/odoo/odoo/pull/179007/files#r1747123097

Reply:
I leave it unresolved like https://github.com/odoo/odoo/pull/179007#discussion_r1747123097 for tech review 😄